lib/Pod/Simple/RTF.pm

```diff
@@ -551,7 +551,7 @@ my $other_unicode =
         Pod::Simple::BlackBox::my_qr('([\x{10000}-\x{10FFFF}])', "\x{10000}");
 
 sub esc_uni($) {
-    use if $] le 5.006002, 'utf8';
+    use if do { no integer; "$]" >= 5.006002 }, 'utf8';
```

Comment:
This is not equivalent to the previous line, but I think what you have here now is correct, as surely one does not mean to `use utf8` on a version where it does not exist! :)

Reply:
Oh wow, that's a bug I introduced myself! This `>=` should indeed be a `<=`. Will push a fixed commit.

Thanks for catching this!